MAUI/DataGrid/export-to-excel.md

```diff
@@ -0,0 +1,548 @@
+option.CanExportGroups = true;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format with group summary rows" src="Images\Export-to-Excel\maui-datagrid-export-group-summary.png" width="689"/>
+
+#### Export table summary
+
+By default, table summaries in the data grid will be exported to Excel. To export the SfDataGrid without table summaries, set the `DataGridExcelExportingOption.CanExportTableSummary` property to `false`.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.CanExportTableSummary= true;
+{% endhighlight %}
+{% endtabs %}
+
+* CanExportTableSummary = true;
+<img alt="Export DataGrid with table summary" src="Images\Export-to-Excel\maui-datagrid-export-table-summary.png" width="689"/>
+
+
+* CanExportTableSummary = false;
+
+<img alt="Export DataGrid without table summary" src="Images\Export-to-Excel\maui-datagrid-exclude-table-summary.png" width="689"/>
+
+
+### Export groups with outlines
+
+To export the data grid with applied grouping, enable the group expand or collapse option in the Excel sheet by setting the `DataGridExcelExportingOption.CanAllowOutlining` to `true`. The default value of this property is false so, you cannot expand or collapse the group in the Excel sheet.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+        option.CanAllowOutlining = true;
+{% endhighlight %}
+{% endtabs %}
+
+### Exclude columns when exporting
+
+By default, all the columns (including hidden columns) in the SfDataGrid will be exported to Excel. To exclude some particular columns when exporting to Excel, add those columns to the `DataGridExcelExportingOption.ExcludeColumns` property in `DataGridExcelExportingOption` list.
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+        var list = new List<string>();
+        list.Add("EmployeeID");
+        list.Add("Name");
+        option.ExcludedColumns = list;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Exclude columns when exporting" src="Images\Export-to-Excel\maui-datagrid-excluded-columns.png" width="689"/>
+
+### Export Selected Rows of SfDataGrid
+
+SfDataGrid allows you to export only the currently selected rows in the grid to the worksheet using the `DataGridExcelExportingController.ExportToExcel` method by passing the instance of the SfDataGrid and `SfDataGrid.SelectedRows` collection as an argument.
+
+Refer the below code to export the selected rows alone to Excel.
+
+{% tabs %}
+{% highlight c# %}
+ObservableCollection<object> selectedItems = dataGrid.SelectedRows;
+var excelEngine = excelExport.ExportToExcel(this.dataGrid, selectedItems);
+{% endhighlight %}
+{% endtabs %}
+
+
+### Customize Exporting Excel Version
+
+The SfDataGrid allows exporting the data to Excel in specific versions by using the `DataGridExcelExportingOption.ExcelVersion` property.
```

Comment:
The SfDataGrid enables the exportation of data to Excel in specific versions through the utilization of the `DataGridExcelExportingOption.ExcelVersion` property.

Reply:
updated